DOCS-1307 - Document how to create playbooks using Mobot (#6984)
SumoLogic/sumologic-documentation: docs/platform-services/automation-service/playbooks/create-playbooks.md

### Create, edit, and modify playbooks using Mobot

Mobot is the AI Playbook Assistant built into the Playbooks editor. Instead of manually wiring up nodes, you describe what you want in plain language, and Mobot proposes a plan, asks clarifying questions, and builds the playbook for you.

#### Prerequisites

Playbooks automate response actions for monitors, Cloud SIEM insights, entities, and Cloud SOAR incidents. Before building a new one, check whether an existing playbook (or one from App Central) already does what you need.

Follow steps 1–5 above to open the Playbooks page, create, name, and describe your playbook. Then:
1. In the chat box, describe the automation you want in plain language. For example, `Create a playbook that creates tickets and sends notifications`.<br/><img src={useBaseUrl('img/cse/prompt-for-mobot.png')} style={{border:'1px solid gray'}} alt="Mobot chatbox" />
1. Answer Mobot's clarifying questions. Mobot checks your org's available integrations, asks which ones you'd like to use, then walks through each action node one at a time to gather the configuration details it needs.

> 1. Review and approve the plan Mobot proposes, including the playbook name, trigger, steps, and flow between them.

Reviewer: No point in reviewing the playbook name, the agent doesn't have the capability of modifying the playbook name. Might confuse the user who tries to edit playbook name while reviewing.

1. Once every step is confirmed, Mobot saves the playbook as a draft and posts a summary table of what was built (Step / Action / Details).
1. To make changes, send a follow-up request describing the edit.<br/><img src={useBaseUrl('img/cse/edit-playbook.png')} style={{border:'1px solid gray'}} alt="edit-playbook" width="200"/><br/>Mobot returns an updated plan reflecting the new flow.
1. Reply `Yes` (or similar) to approve, and Mobot rebuilds and resaves the playbook.
1. Click any node on the canvas to verify the details Mobot filled in, such as the integration, resource, and field mappings pulled from the trigger payload.<br/><img src={useBaseUrl('img/cse/playbook-flow.png')} style={{border:'1px solid gray'}} alt="playbook-flow"/>

> 1. Click **Publish** at the bottom of the editor to make the playbook available for use in automations.

Reviewer 2: Publish has been moved to the header. We need to use the updated screenshot here

#### Prerequisites

Playbooks automate response actions for monitors, Cloud SIEM insights, entities, and Cloud SOAR incidents. Before building a new one, check whether an existing playbook (or one from App Central) already does what you need.

Follow the steps below to create playbooks via Mobot:
1. [**New UI**](/docs/get-started/sumo-logic-ui). In the main Sumo Logic menu, select **Automation > Playbooks**. You can also click the **Go To...** menu at the top of the screen and select **Playbooks**.  <br/>[**Classic UI**](/docs/get-started/sumo-logic-ui-classic).  In the main Sumo Logic menu, select **Automation**. <br/>Previously-created playbooks display.
1. Click the **+ Create Playbook** button.
1. Click the untitled playbook and rename it.<br/><img src={useBaseUrl('img/cse/automations-new-playbook-dialog.png')} style={{border:'1px solid gray'}} alt="New playbook dialog" />
1. Enter a **Description** of the playbook to help others understand how to use it.

> 1. Select the incident **Type**. (For example, for Cloud SIEM automations, select **Cloud SIEM**. For playbooks run from inside another playbook, you can select another incident type to associate with it, for example, **Denial of Service**, **Malware**, **Phishing**, and so on.)

Reviewer 3: These five steps (navigate to Playbooks, click **+ Create Playbook**, rename, enter Description, select Type) are identical to steps 1–5 in the "Create a new playbook" procedure right above this one, down to reusing the same screenshot. That's a two-places-to-update problem — the same issue the sibling PR (#6981) fixed for the monitors doc by trimming duplicated content instead of repeating it.

Suggest replacing steps 1–5 here with something like: "Follow steps 1–5 above to open the Playbooks page, create, name, and describe your playbook. Then:" and picking up directly at the chat-box step. That way a future change to the shared creation flow only needs to be updated once.

> 1. In the chat box, describe the automation you want in plain language. For example, Create a playbook that creates tickets and sends notifications.<br/><img src={useBaseUrl('img/cse/prompt-for-mobot.png')} style={{border:'1px solid gray'}} alt="Mobot chatbox" />

Reviewer 3: Minor formatting consistency: the companion Mobot-monitors doc wraps literal example prompts in inline code (e.g. `` `Watch my checkout latency` ``). Suggest doing the same here.

```suggestion
1. In the chat box, describe the automation you want in plain language. For example, `Create a playbook that creates tickets and sends notifications`.<br/><img src={useBaseUrl('img/cse/prompt-for-mobot.png')} style={{border:'1px solid gray'}} alt="Mobot chatbox" />
```

1. Answer Mobot's clarifying questions. Mobot checks your org's available integrations, asks which ones you'd like to use, then walks through each action node one at a time to gather the configuration details it needs.
1. Review and approve the plan Mobot proposes, including the playbook name, trigger, steps, and flow between them.
1. Once every step is confirmed, Mobot saves the playbook as a draft and posts a summary table of what was built (Step / Action / Details).
1. To make changes, send a follow-up request describing the edit.<br/><img src={useBaseUrl('img/cse/edit-playbook.png')} style={{border:'1px solid gray'}} alt="edit-playbook" width="200"/><br/>Mobot returns an updated plan reflecting the new flow.

> 1. Reply **yes it looks good** to approve, and Mobot rebuilds and resaves the playbook.

Reviewer 3: Same formatting note — the monitors doc uses inline code for literal replies ("Reply `Yes` or `Proceed` to deploy"). Also, requiring the exact phrase "yes it looks good" seems narrower than what Mobot likely accepts.

```suggestion
1. Reply `Yes` (or similar) to approve, and Mobot rebuilds and resaves the playbook.
```
